CMakePCHCompiler.cmake

```diff
@@ -51,7 +51,7 @@ function(target_precompiled_header) # target [...] header
 	if(ARGS_REUSE AND NOT TARGET "${ARGS_REUSE}")
 		message(SEND_ERROR "Re-use target \"${ARGS_REUSE}\" does not exist.")
 		return()
-	endif()
+	endif()	
```

Comment:
Trailing space?

Reply:
No? Is that the last line? Just something an editor did. Or a stray key press.